components-core/src/main/java/com/adyen/checkout/components/model/payments/request/EContextPaymentMethod.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ozgur on 7/6/2022.
+ */
+
+package com.adyen.checkout.components.model.payments.request
+
+abstract class EContextPaymentMethod : PaymentMethodDetails() {
+
+    abstract var firstName: String?
+    abstract var lastName: String?
+    abstract var telephoneNumber: String?
+    abstract var shopperEmail: String?
+
+    companion object {
+        const val FIRST_NAME = "firstName"
```

Comment:
Should these be `protected`?

Reply:
It doesn't work very straightforwardly. We have to mark it with `@JvmStatic` to be able to make it protected but then we have to remove the `const` because they're not supported together. Leaving as it is.

Reply:
Ahhh i forgot it's in the companion, nevermind 👍